src/amp-story-player/amp-story-player-impl.js

```diff
@@ -309,9 +315,21 @@ export class AmpStoryPlayer {
     this.initializeIframes_();
     this.initializeButton_();
     this.signalReady_();
+    this.setUpAutoplayClickShield_();
```

Comment:
Can we set this up only when the `preview` attribute is present?

Reply:
Done!